packages/components/src/components/data-table/_data-table-sort.scss

```diff
@@ -18,7 +18,7 @@
   // -------------------------------------
   .#{$prefix}--data-table--sort th,
   .#{$prefix}--data-table--sort
-    th:first-of-type:not(.#{$prefix}--table-column-checkbox) {
+    th:first-of-type:not(.#{$prefix}--table-column-checkbox):not(.#{$prefix}--table-expand) {
     padding: 0;
```

Comment:
it looks like this rule is causing the expand all chevron to be misaligned (but only in Firefox)
![image](https://user-images.githubusercontent.com/8265238/57930763-65c1fa80-787c-11e9-8945-1dbf6e56518c.png)

the expand all chevron needs a top padding of `.875rem` like the other chevrons

Reply:
Good catch @emyarod - Fixed.